lib/properties/energy.dart

```diff
@@ -47,6 +49,10 @@ class Energy extends Property<ENERGY, double> {
         coefficientProduct: 4.1867999409,
         name: ENERGY.calories,
       ),
+      Node(
+        coefficientProduct: 4186.7999409,
+        name: ENERGY.kilocalories,
+      ),
```

Comment:
I would have written 1 kilocalories as 1000 calories, I think it's more readable and if we miss some figures in the decimal part we have to fix just one line (instead of two)